.github/workflows/sandbox_register.yml

```diff
@@ -22,7 +22,15 @@ jobs:
         with:
           fetch-depth: "0"
       - uses: unionai/flytectl-setup-action@v0.0.1
-      - name: setup sandbox
-        run: flytectl sandbox start --sourcesPath=$GITHUB_WORKSPACE
-      - name: fast register
+      - name: Setup sandbox
+        run: make setup
+      - name: Setup flytectl config
+        run: eval $(make flytectl-config)
```

Comment:
does this not set `FLYTECTL_CONFIG`? what does it do then?

Reply:
yes